core/org.osate.annexsupport/src/org/osate/annexsupport/AnnexValidator.java

```diff
@@ -1,126 +1,126 @@
-package org.osate.annexsupport;

```

Comment:
What happened with this file?

Reply:
Not really sure, it was showing changes although none had occurred.  Fixed it now though.